package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "rdf-js",
+  "version": "4.0.1",
+  "types": "index.d.ts",
+  "author": {
+    "name": "RDF/JS Representation Task Force",
+    "url": "https://github.com/rdfjs/representation-task-force",
+    "email": "public-rdfjs@w3.org"
+  },
+  "scripts": {
+    "lint": "exit 0",
+    "test": "tsc --noEmit"
+  },
+  "dependencies": {
+    "@types/node": "*"
+  },
+  "devDependencies": {
+    "typescript": "^4.0.3"
+  },
+  "maintainers": [{
+    "name": "Ruben Taelman",
+    "url": "https://github.com/rubensworks"
+  }, {
+    "name": "Laurens Rietveld",
+    "url": "https://github.com/LaurensRietveld"
+  }, {
+    "name": "Tomasz Pluskiewicz",
+    "url": "https://github.com/tpluscode"
+  }],
```

Comment:
Current authors from DT as maintainers

Reply:
Just to simplify things going forward I would suggest we remove this from package.json. This repository will serve as issue tracker and original authorship is retained in git log and comments.

Reply:
I'm fine with removal.